config/image-resize.py

```diff
@@ -0,0 +1,54 @@
+from PIL import Image
+import base64
+from io import BytesIO
+import logging
+
+
+def resize_image(base64_image_str, size, maintain_aspect_ratio=False):
+    """
+    Resize a base64 encoded image to the specified size.
+
+    :param base64_image_str: Base64 encoded image string
+    :param size: Tuple (width, height) specifying the new size
```

Comment:
Suggest adding maintain_aspect_ratio here, and setting to True by default, since I suspect this will be more commonly desired.